bat/ASM.BAT

```diff
@@ -15,9 +15,18 @@ if /I %1. == . (set /P z_file=Hit Enter for help or suppply name of mlc file to
         ) else (set z_file=%1)
 if /I %z_file%. == . goto help
 :chkfile
-if not exist %z_file%.MLC (echo %0 ERROR: %z_file%.MLC was not found
-                           set z_ReturnCode=16
-                           goto return)
+rem convert Unix path notation to Windows before checking the file
```

Comment:
Why? This is a Windows BAT file. Shouldn't it be a "JCL error" if a Windows user enters *nix file separators? Is there some reason for doing this?